dbms/src/TestUtils/mockExecutor.h

```diff
@@ -117,8 +122,9 @@ class DAGRequestBuilder
 class MockDAGRequestContext
 {
 public:
-    explicit MockDAGRequestContext(Context context_)
+    explicit MockDAGRequestContext(Context context_, Int32 collation_ = TiDB::ITiDBCollator::UTF8MB4_BIN)
```

Comment:
Why is the default parameter not `no collation`?

Reply:
> Why is the default parameter not `no collation`?

Because tidb opens the collation by default recently, and sets it as UTF8MB4_BIN.